react/setupTests.js

```diff
@@ -0,0 +1,7 @@
+global.navigator.geolocation = {
+  getCurrentPosition: jest.fn(),
+}
+
+global.navigator.permissions = {
+  query: jest.fn(),
+}
```

Comment:
I wish it could work like this:

```suggestion
global.navigator = {
  geolocation: {
  	getCurrentPosition: jest.fn(),
  },
  permissions: {
	query: jest.fn(),
  }
}
```

but it simply makes `geolocation` and `permissions` undefined 🤷🏾 
no big deal, but if anyone knows why this doesn't works, I would like to know

Reply:
Did you try something like this

```
global.navigator = () => {
  const geolocation = {
    getCurrentPosition: jest.fn(),
  }

  const permissions = {
    query: jest.fn(),
  }

  return {
    geolocation,
    permissions
  }
}
```

Reply:
no but hmm, let me try

Reply:
didn't worked, same problem:

`TypeError: Cannot read property 'query' of undefined` (same for `getCurrentPosition`)

I think it kind of makes sense, because navigator isn't a function

Reply:
And like that? 🤔 

```
global.navigator = (() => {
  const geolocation = {
    getCurrentPosition: jest.fn(),
  }

  const permissions = {
    query: jest.fn(),
  }

  return {
    geolocation,
    permissions
  }
}())
```

Reply:
But, personally, I don't see a problem in keeping it the way you did.

Reply:
I was really thinking that this one would work, but nope. Same error :thonking:

Reply:
shouldn't this file be `setupTestsAfterEnv.js`?